test/parse-urn.spec.ts

```diff
@@ -11,78 +19,82 @@ function testValidUrnToInclude(urn: string, toInclude: Partial<DecentralandAsset
+  describe('land urns', () => {
+    it('Print LAND resolution for readme', async () => {
+      expect(await parseUrn('urn:decentraland:sepolia:LAND:-10,-13?atBlock=151231111')).toMatchObject({
+        blockchain: 'ethereum',
+        contractAddress: '0x42f4ba48791e2de32f5fbf553441c2672864bb33',
+        id: '0xfffffffffffffffffffffffffffffff6fffffffffffffffffffffffffffffff3',
+        namespace: 'decentraland',
+        network: 'sepolia',
+        type: 'blockchain-asset',
+        x: -10,
+        y: -13
+      })
     })
-  })
 
-  it('test land', async () => {
-    expect(
-      await resolveEthereumAsset(new URL('urn:decentraland:ethereum:LANDPROXY:0x1'), {
-        contract: 'LANDPROXY',
-        network: 'ethereum',
-        tokenId: '0x1'
+    it('test land', async () => {
+      expect(
+        await resolveEthereumAsset(new URL('urn:decentraland:ethereum:LANDPROXY:0x1'), {
+          contract: 'LANDPROXY',
+          network: 'ethereum',
+          tokenId: '0x1'
+        })
+      ).toMatchObject({
+        contractAddress: '0xf87e31492faf9a91b02ee0deaad50d51d56d5d4d',
+        network: 'mainnet',
+        blockchain: 'ethereum',
+        type: 'blockchain-asset',
+        id: '0x1'
       })
-    ).toMatchObject({
-      contractAddress: '0xf87e31492faf9a91b02ee0deaad50d51d56d5d4d',
-      network: 'mainnet',
-      blockchain: 'ethereum',
-      type: 'blockchain-asset',
-      id: '0x1'
     })
-  })
 
-  it('test land (query string)', async () => {
-    const t = await parseUrn('urn:decentraland:sepolia:LAND:0x1?atBlock=151231111#4')
-    expect(t).toHaveProperty('uri')
-    expect(t.uri.toString()).toEqual('urn:decentraland:sepolia:LAND:0x1?atBlock=151231111#4')
-  })
-
-  // it("test land (address)", async () => {
-  //   expect(await parseUrn("urn:decentraland:ethereum:0xf87e31492faf9a91b02ee0deaad50d51d56d5d4d:0x1")).toMatchObject({
-  //     contractAddress: "0xf87e31492faf9a91b02ee0deaad50d51d56d5d4d",
-  //     type: "blockchain-asset",
-  //     network: "mainnet",
-  //     id: "0x1",
-  //   })
-  // })
+    it('test land (query string)', async () => {
+      const t = await parseUrn('urn:decentraland:sepolia:LAND:0x1?atBlock=151231111#4')
+      expect(t).toHaveProperty('uri')
+      expect(t.uri.toString()).toEqual('urn:decentraland:sepolia:LAND:0x1?atBlock=151231111#4')
+    })
 
-  it('test land (sepolia)', async () => {
-    expect(await parseUrn('urn:decentraland:sepolia:LAND:0x1')).toMatchObject({
-      contractAddress: '0x42f4ba48791e2de32f5fbf553441c2672864bb33',
-      blockchain: 'ethereum',
-      network: 'sepolia',
-      id: '0x1'
+    // it("test land (address)", async () => {
+    //   expect(await parseUrn("urn:decentraland:ethereum:0xf87e31492faf9a91b02ee0deaad50d51d56d5d4d:0x1")).toMatchObject({
+    //     contractAddress: "0xf87e31492faf9a91b02ee0deaad50d51d56d5d4d",
+    //     type: "blockchain-asset",
+    //     network: "mainnet",
+    //     id: "0x1",
+    //   })
+    // })
```

Comment:
Can we remove this test? it was commented out 3 years ago and it doesn't seem we are going to recover it.